server/storage/players.lua

```diff
@@ -230,8 +230,8 @@ end
 ---@param tableName string
 ---@return boolean
 local function doesTableExist(tableName)
-    local tbl = MySQL.single.await(('SELECT COUNT(*) FROM information_schema.TABLES WHERE TABLE_NAME = \'%s\' AND TABLE_SCHEMA in (SELECT DATABASE())'):format(tableName))
-    return tbl['COUNT(*)'] > 0
+    local result = MySQL.single.await(('SELECT COUNT(*) as count FROM information_schema.TABLES WHERE TABLE_NAME = \'%s\' AND TABLE_SCHEMA in (SELECT DATABASE())'):format(tableName))
```

Comment:
I think the best thing here might be to select scalar 1 then return not not result. Also, we should use placeholders '?' rather than string format